operations/research-and-development/product/development-process/mobile-feature-guidelines.md

```diff
@@ -11,6 +11,7 @@ While not every web app feature needs parity with mobile, this is a discussion t
 - If a feature requires mobile code changes, the Mobile team PM (Eric Sethna) should be made aware of the feature before it’s committed to the feature team's quarterly roadmap. Please involve the Mobile team PM in roadmap planning and quarterly OKR planning discussions. The same applies even if the feature team will be doing the mobile implementation themselves, given that the Mobile team will be involved in code reviews, testing, and should provide guidance on the feature implementation.
 - During design and spec development of any feature that will involve mobile code changes, please involve the Mobile PM (Eric Sethna) and Mobile dev lead (Elias Nahum) at all stakeholder checkpoints including requirements gathering, design option exploration, and final design review.
 - Set deliverables taking into account the limitation that a team may not have enough skills to develop features for mobile. Consider the impact if the feature is left out from mobile, document the reasons, and set goals to cover the mobile space. The feature should be [released as beta or experimental](https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#can-we-take-this-feature-out-of-beta) and any documentation should clearly communicate the impact of not having mobile support as well as when mobile support is expected.
+- If a new feature is related to Single Sign-On (SSO), plan to release the feature on Mobile Apps prior to Cloud to avoid users from being blocked from using the Mobile Apps.
```

Comment:
I would generalize this even more to say anything related to authentication (ie if a user would be unable to login into the mobile app without the changes being present)